SidecarSubtitles/BasicSidecarSubtitlesPlayer/objc/BasicSidecarSubtitlesPlayer/ViewController.m

```diff
@@ -89,6 +96,18 @@ - (void)requestContentFromPlaybackService
     }];
 }
 
+- (NSArray *)setupSubtitles
+{
+    NSArray *textTracks = @[@{ kBCOVSSTextTracksKeyKind: kBCOVSSTextTracksKindSubtitles,
+                               kBCOVSSTextTracksKeyLabel: @"English",
+                               kBCOVSSTextTracksKeyDefault: @YES,
+                               kBCOVSSTextTracksKeySourceLanguage: @"en",
+                               kBCOVSSTextTracksKeySource: @"Add your VTT caption file URL here",
```

Comment:
None that fit the video content.
There is a sample VTT file that I tested with, as well as one of my own, but those are either:
1. Too short for the content (the sample caption file)
2. Not appropriate for the content (my test file, which is just a bunch of numbers).